Easydict/Feature/Service/OpenAI/EZOpenAIService.m

```diff
@@ -62,6 +62,14 @@ - (NSString *)model {
     return model;
 }
 
+- (void)setModel:(NSString *)model {
+    [[NSUserDefaults standardUserDefaults] setValue:model forKey:EZOpenAIModelKey];
+}
+
+- (NSArray<NSString *> *)availableModels {
+    return @[@"gpt-3.5-turbo-0125", @"gpt-4-0125-preview"];
```

Comment:
Can we optimize this code? It duplicates the code in the configuration, and these models will follow OpenAI changes later.

```swift
enum OpenAIModels: String, CaseIterable {
    case gpt3_5_turbo_0125 = "gpt-3.5-turbo-0125"
    case gpt4_0125_preview = "gpt-4-0125-preview"
}
```

Reply:
You're right, I have rewritten the `OpenAIService` to reuse this enum type.

Reply:
ok